src/index.js

```diff
@@ -1,11 +1,16 @@
+/* eslint-disable linebreak-style */
```

Comment:
- [ ] Please fix all the linter errors in your entire codebase. Linters ensure that you implement programming best practices by making a flag on some programmatic/stylistic errors. Do not disable them.